nucypher/config/keyring.py

```diff
@@ -422,13 +422,6 @@ def _generate_key_filepaths(public_key_dir: str,
 
         return __key_filepaths
 
-    def _export_wallet_to_node(self, blockchain, password):  # TODO: Deprecate with geth.parity signing EIPs
```

Comment:
Is this TODO addressed in this PR, too?

Reply:
It's not addressed. The `_export_wallet_to_node` method is unused and the TODO suggests that the method should be deprecated, so I thought I might as well remove it.